### ISSUE TITLE
/DeepSpeech/examples/tiny# sh run_train.sh 运行时出错

### ISSUE THREAD

**Issue by JoeBlack220** [PaddlePaddle/DeepSpeech#64]
root@3328a2284d27:/DeepSpeech/examples/tiny# sh run_train.sh
----------- Configuration Arguments -----------
augment_conf_path: conf/augmentation.config
batch_size: 16
dev_manifest: data/tiny/manifest.tiny
init_model_path: None
is_local: 1
learning_rate: 1e-05
max_duration: 27.0
mean_std_path: data/tiny/mean_std.npz
min_duration: 0.0
num_conv_layers: 2
num_iter_print: 100
num_passes: 20
num_proc_data: 1
num_rnn_layers: 3
output_model_dir: ./checkpoints/tiny
rnn_layer_size: 2048
share_rnn_weights: 1
shuffle_method: batch_shuffle_clipped
specgram_type: linear
test_off: 0
train_manifest: data/tiny/manifest.tiny
trainer_count: 8
use_gpu: 1
use_gru: 0
use_sortagrad: 1
vocab_path: data/tiny/vocab.txt

I1206 11:44:56.762332 47 Util.cpp:166] commandline: --use_gpu=1 --rnn_use_batch=True --log_clipping=True --trainer_count=8
F1206 11:44:57.226635 47 hl_cuda_cublas.cc:137] Check failed: CUBLAS_STATUS_SUCCESS == g_cublasStat (0 vs. 1) Cublas Error: [cublas status]: not initialized [cublas init] Cublas create handle faild!
*** Check failure stack trace: ***
@ 0x7f7b10b7f04d google::LogMessage::Fail()
@ 0x7f7b10b81398 google::LogMessage::SendToLog()
@ 0x7f7b10b7eb5b google::LogMessage::Flush()
@ 0x7f7b10b8226e google::LogMessageFatal::~LogMessageFatal()
@ 0x7f7b10b1d7a1 hl_cublas_init()
@ 0x7f7b10b2a6e4 hl_create_global_resources()
@ 0x7f7b10b2b1a4 hl_specify_devices_start()
@ 0x7f7b10b2b47d hl_start()
@ 0x7f7b10aace7e paddle::initMain()
@ 0x7f7b10b65621 initPaddle()
@ 0x7f7b10601347 _wrap_initPaddle
@ 0x4c468a PyEval_EvalFrameEx
@ 0x4c9d8f PyEval_EvalFrameEx
@ 0x4c2765 PyEval_EvalCodeEx
@ 0x4de6fe (unknown)
@ 0x4b0cb3 PyObject_Call
@ 0x4c6ad1 PyEval_EvalFrameEx
@ 0x4c2765 PyEval_EvalCodeEx
@ 0x4ca8d1 PyEval_EvalFrameEx
@ 0x4c2765 PyEval_EvalCodeEx
@ 0x4ca8d1 PyEval_EvalFrameEx
@ 0x4c2765 PyEval_EvalCodeEx
@ 0x4c2509 PyEval_EvalCode
@ 0x4f1def (unknown)
@ 0x4ec652 PyRun_FileExFlags
@ 0x4eae31 PyRun_SimpleFileExFlags
@ 0x49e14a Py_Main
@ 0x7f7b35555830 __libc_start_main
@ 0x49d9d9 _start
@ (nil) (unknown)
Aborted (core dumped)
Fail in training!

**Comment by john81529** [comment]
你这是内存问题，可以考虑下减少batch的size,降低train counter的数值

**Comment by kuke** [comment]
@JoeBlack220 ```trainer_count```要设置成你实际可用的GPU数目，另如果你用的还是老的Docker镜像，请用新的替代: ```paddlepaddle/deep_speech:latest-gpu```。

**Comment by JoeBlack220** [comment]
谢谢，减少batch_size和 trainer_count之后可以训练了！

**Comment by fangmingbnu** [comment]
-----------  Configuration Arguments -----------
augment_conf_path: conf/augmentation.config
batch_size: 16
dev_manifest: data/tiny/manifest.tiny
init_model_path: None
is_local: 1
learning_rate: 1e-05
max_duration: 27.0
mean_std_path: data/tiny/mean_std.npz
min_duration: 0.0
num_conv_layers: 2
num_iter_print: 100
num_passes: 2
num_proc_data: 3
num_rnn_layers: 3
output_model_dir: ./checkpoints/tiny
rnn_layer_size: 2048
share_rnn_weights: 1
shuffle_method: batch_shuffle_clipped
specgram_type: linear
test_off: 0
train_manifest: data/tiny/manifest.tiny
trainer_count: 1
use_gpu: 1
use_gru: 0
use_sortagrad: 1
vocab_path: data/tiny/vocab.txt
------------------------------------------------
I0321 17:45:16.130343 14602 Util.cpp:166] commandline:  --use_gpu=1 --rnn_use_batch=True --log_clipping=True --trainer_count=1 
F0321 17:45:16.943452 14602 hl_cuda_cublas.cc:137] Check failed: CUBLAS_STATUS_SUCCESS == g_cublasStat (0 vs. 1) Cublas Error: [cublas status]: not initialized [cublas init] Cublas create handle faild!
*** Check failure stack trace: ***
    @     0x7fa9fb4340bd  google::LogMessage::Fail()
    @     0x7fa9fb436408  google::LogMessage::SendToLog()
    @     0x7fa9fb433bcb  google::LogMessage::Flush()
    @     0x7fa9fb4372de  google::LogMessageFatal::~LogMessageFatal()
    @     0x7fa9fb3d2861  hl_cublas_init()
    @     0x7fa9fb3df7a4  hl_create_global_resources()
    @     0x7fa9fb3e0264  hl_specify_devices_start()
    @     0x7fa9fb3e053d  hl_start()
    @     0x7fa9fb361a6e  paddle::initMain()
    @     0x7fa9fb41a691  initPaddle()
    @     0x7fa9faeac4b7  _wrap_initPaddle
    @           0x4bc3fa  PyEval_EvalFrameEx
    @           0x4c136f  PyEval_EvalFrameEx
    @           0x4b9ab6  PyEval_EvalCodeEx
    @           0x4d54b9  (unknown)
    @           0x4a577e  PyObject_Call
    @           0x4bed3d  PyEval_EvalFrameEx
    @           0x4b9ab6  PyEval_EvalCodeEx
    @           0x4c1e6f  PyEval_EvalFrameEx
    @           0x4b9ab6  PyEval_EvalCodeEx
    @           0x4c1e6f  PyEval_EvalFrameEx
    @           0x4b9ab6  PyEval_EvalCodeEx
    @           0x4eb30f  (unknown)
    @           0x4e5422  PyRun_FileExFlags
    @           0x4e3cd6  PyRun_SimpleFileExFlags
    @           0x493ae2  Py_Main
    @     0x7faa34f75830  __libc_start_main
    @           0x4933e9  _start
    @              (nil)  (unknown)
Aborted (core dumped)
Fail in training!


我也有一样的问题，但是我已经减少了batch_size和 trainer_count，但是还是有这个问题，求助大家！
